packages/beacon-node/src/api/impl/utils.ts

```diff
@@ -0,0 +1,25 @@
+import {ApiError} from "./errors.js";
+
+/**
+ * Ensures that the array contains unique values, and throws an ApiError
+ * otherwise.
+ * @param array - The array to check for uniqueness.
+ * @param message - The message to put in the ApiError if the array contains
+ * duplicates.
+ */
+export function ensureUniqueItemsOrThrow(array: unknown[] | undefined, message: string): void {
```

Comment:
can we rename this please (as suggested earlier)

```suggestion
export function assertUniqueItems(array: unknown[] | undefined, message: string): void {
```